packages/clay-navigation/src/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import classNames from 'classnames';
+import React from 'react';
+
+export interface IBreadcrumbItem {
+	active?: boolean;
+
```

Comment:
no need for line breaks here unless you are adding jsdocs

Reply:
Solved!